release-notes/opensearch-release-notes-1.0.0-rc1.md

```diff
@@ -0,0 +1,324 @@
+
+
+## ENHANCEMENTS
+
+### Opensearch Anomaly Detection Dashboards
+* Update sample data naming conventions ([#24](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/24))
+* Update API call endpoints from _opendistro to _plugins ([#25](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/25))
+
+
+### Opensearch Dashboards Notebooks
+* Add output only parameter to Context Menu Base URL ([#18](https://github.com/opensearch-project/dashboards-notebooks/pull/18))
+* Add Query Parameter for Selected View ([#17](https://github.com/opensearch-project/dashboards-notebooks/pull/17))
+
+
+### Opensearch Dashboards Notebooks
+* Add output only parameter to Context Menu Base URL ([#18](https://github.com/opensearch-project/dashboards-notebooks/pull/18))
+* Add Query Parameter for Selected View ([#17](https://github.com/opensearch-project/dashboards-notebooks/pull/17))
+
+
+### Opensearch Dashboards Reports
+* Use output_only parameter for notebook reports ([#32](https://github.com/opensearch-project/dashboards-reports/pull/32))
+* Add Logic to Auto-populate Notebooks from Context menu ([#30](https://github.com/opensearch-project/dashboards-reports/pull/30))
+* [Query Builder] Correctly handle matched phrases when a single value is specified or when the match phrases is negated ([#33](https://github.com/opensearch-project/dashboards-reports/pull/33))
+* Replace osd header in context menu ([#37](https://github.com/opensearch-project/dashboards-reports/pull/37))
+* Remove visualization editor in visualization reports ([#50](https://github.com/opensearch-project/dashboards-reports/pull/50))
+* Increase chromium timeout to 100s ([#58](https://github.com/opensearch-project/dashboards-reports/pull/58))
+
+
+### Opensearch Index Management
+* Improve integration with data streams ([#13](https://github.com/opensearch-project/index-management/pull/13))
+
+
+### Opensearch k-NN
+* Add extra place to increase k-NN graph query errors ([#26](https://github.com/opensearch-project/k-NN/pull/26))
+
+
+### Opensearch Performance Analyzer
+* Add CONFIG_DIR_NOT_FOUND in StatExceptionCode ([#11](https://github.com/opensearch-project/performance-analyzer-rca/pull/11))
+
+
+### Opensearch Perftop
+* Add env.js to store global property ([#10](https://github.com/opensearch-project/perftop/pull/10))
+
+
+### Opensearch Security
+* Allow attempt to load security config in case of plugin restart even if security index already exists ([#1154](https://github.com/opensearch-project/security/pull/1154))
+* Allowing granular access for data-stream related transport actions ([#1170](https://github.com/opensearch-project/security/pull/1170))
+
+
+### Opensearch Security Dashboards Plugin
+* Add svg content [#766](https://github.com/opensearch-project/security-dashboards-plugin/pull/766)
+* Changes regarding the anonymous auth flow [#768](https://github.com/opensearch-project/security-dashboards-plugin/pull/768)
```

Comment:
PR info not enclosed in (), not following the same standard as others.

Reply:
Added. Thanks @sreekarjami